common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -96,6 +96,16 @@ function (HTML5Video, Resizer) {
             $(window).on('unload', state.saveState);
 
             if (!state.isFlashMode() && state.speed != '1.0') {
+
+                // Work around a bug in the Youtube API that causes videos to
```

Comment:
Do we have a link to the bug report? It would be useful to put it here for tracking the status of the issue.